changelog.d/11284.feature

```diff
@@ -0,0 +1 @@
+Include bundled relations in response to the `/relations` API, per [MSC3440](https://github.com/matrix-org/matrix-doc/pull/3440).
```

Comment:
to be clear, we're talking about relations of relations of the original event, right? 

Suppose I request the relations for event E, where:
 * E has relations A, B and C
 * each of A, B and C has its own relations X, Y and Z respectively
 * once this PR lands, we will bundle X, Y and Z in with A, B and C, whereas we would not before.

In other words, we're considering meta-relations...

Is that a correct understanding? If so, I think both this newsfile and the PR summary could be clearer.

Reply:
> Is that a correct understanding? If so, I think both this newsfile and the PR summary could be clearer.

Yes, this is correct. I've been having trouble with some of the wording around relations. Do you have any suggestions of how to make it clearer?

Reply:
For the newsfile:

> When returning relation events from the `/relations` API, bundle any relations of those relations into the result, per MSC3440.

It's not great, but it's reasonably accurate.

For the PR summary:

> Bundle relations of relations into the `/relations` result.

Reply:
Note that this was also updated in MSC2675. It now reads:

> The following client-server APIs should bundle aggregations with events they return:
>
> * ...
> * `GET /relations`, as proposed in this MSC.

```suggestion
When returning relation events from the `/relations` API, bundle any relations of those relations into the result, per updates to [MSC2675](https://github.com/matrix-org/matrix-doc/pull/2675).
```